rhwp/rhwp-studio/src/ui/agent-sidebar/agent-sidebar.css

```
@@ -6507,3 +6507,334 @@ html.desktop-mac .ag-fullscreen .ag-environment-panel {
+
+.ag-question-option:disabled {
+  cursor: default;
+  opacity: .64;
+}
+
+.ag-question-option-number {
+  display: grid;
+  place-items: center;
+  width: 22px;
+  height: 22px;
+  border: 1px solid currentColor;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)